.circleci/config.yml

```diff
@@ -1768,10 +1770,11 @@ jobs:
       - run:
           name: Modify Payer account for Public testnet start from saved state test
           command: |
-            echo -n "$KEY_DevTestNetTreasury" > /repo/test-clients/src/main/resource/StartUpAccount.txt;
+            echo -n "$KEY_DevTestNetTreasury";    echo -n /repo/test-clients/src/main/resource/StartUpAccount.txt;
```

Comment:
Reason why `echo -n ` is added to `/repo/test-clients/src/main/resource/StartUpAccount.txt;`

Reply:
Just want this to be a reminder to remember change it back to update the startup account when it starts from saved state. Right now, since it's from scratch, so only default account works.

Reply:
Then I think we can comment out both of them and add a comment it should be enabled once tests are moved to start from state.